R/utils-varnish.R

```diff
@@ -83,8 +83,8 @@ set_globals <- function(path) {
   # Resources
   learner <- create_resources_dropdown(these_resources[["learners"]],
     "learners")
-  instructor <- create_resources_dropdown(these_resources[["instructors"]],
-    "instructors")
+  instructor_all <- c(these_resources$instructors, these_resources$learners)
+  instructor <- create_resources_dropdown(instructor_all, "instructors")
```

Comment:
I think instead it would be better to combine the output of the learner and instructor resources after the dropdown list is created, separating with a horizontal rule (which I believe we may be able to add a "half-width" class to to stylize it a bit better).

```suggestion
  instructor <- create_resources_dropdown(these_resources[["instructors"]], "instructors")
  instructor$extras <- c(instructor$extras, "<hr>", learner$extras)
  instructor$resources <- c(instructor$resources, "<hr>", learner$extras)
```

Reply:
Note: this change should produce something that looks like this:

![screenshot of dropdown menu showing two sections delineated by a horizontal rule](https://github.com/carpentries/sandpaper/assets/3639446/a45b8791-7c32-4a92-b82d-161a6a89ffc3)